.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -337,6 +307,36 @@ jobs:
             deploy-l2-contracts.out
           retention-days: 7
 
+  update-loadbalancer:
+    needs:
+      - build
+      - deploy
+      - check-obscuro-is-healthy
```

Comment:
Does it need these three or just `check-obscuro-is-healthy` ?

Reply:
I had come across some opinionated view, whether to minimize or be explicit on these and leave it in and had usually done latter, (but perhaps that only makes sense in LARGE workflows to not need to trace job dependencies), you have motivated me to remove for the cleaner style. ⚡ 👍